site/content/integrate/slash-commands/_index.md

```diff
@@ -24,24 +24,26 @@ If you configured a slash command with a trigger word of `test`, a request URL o
 to your Town Square channel, your endpoint will receive:
 
 ```http
-POST /slash-command HTTP/1.1
```

Comment:
Should we keep the first four lines as-is, or okay to change based on the example you shared?

Reply:
Looks fine with everything changed.